### PR TITLE
add support for suse linux

### PR DESCRIPTION
**Pull request by skylerto**
Signed-off-by: skylerl <skylerl@indellient.com>

### Description

Add support for SUSE Linux

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing (paid linux).
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 recipe 'jenkins::master', 'Installs a Jenkins master'
 
-%w(ubuntu debian redhat centos scientific oracle amazon).each do |os|
+%w(ubuntu debian redhat centos scientific oracle amazon suse).each do |os|
```
Comment (tas50) [inline metadata.rb]: This should probably include opensuseleap since suse is just SLES

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Installs and configures Jenkins CI master & node slaves. Resource providers to s
 
 - Debian 7+ (Package installs require 9+ due to dependencies)
 - Ubuntu 14.04+ (Package installs require 16.04+ due to dependencies)
-- RHEL/CentOS/Scientific/Oracle 6+
+- RHEL/CentOS/SUSE/Scientific/Oracle 6+
```
Comment (tas50) [inline README.md]: SUSE should go on its own line since the 6+ part doesn't apply to SUSE

Comment (tas50) [reply README.md]: Also you should clarify what SUSE we're talking about here. SLES or openSUSE

File: metadata.rb (hunk 1 above)
Comment (skylerto) [reply metadata.rb]: We're using SLES specifically in our fork, I can totally modify this to add opensuse support if need be.

File: README.md (hunk 2 above)
Comment (skylerto) [reply README.md]: Whoops! This was totally my bad, I'll push this up.

File: metadata.rb (hunk 1 above)
Comment (skylerto) [reply metadata.rb]: adding test kitchen etc.

**Comment by tas50** [comment]
@skylerto One more question here. How does this actually work with the runit stuff since there's no runit package for SLES?

**Comment by skylerto** [comment]
@tas50 good point! I think that's handled by our wrapper cookbook, let me get those changes in and submit a clean PR.

**Pull request closed without merging**